Automattic/wp-calypso
# Date Range Picker: Refactor the calendar to a standalone component

Related to https://github.com/Automattic/red-team/issues/159

## Proposed Changes

* Extract date range picker to a standalone component
* Add typing for the component

## Why are these changes being made?

* Make the date range picker reusable

## Testing Instructions

* Run Jetpack Cloud locally `yarn start-jetpack-cloud`
* Open a site on Jetpack cloud with Complete plan `http://jetpack.cloud.localhost:3000/activity-log/:siteSlug`
* Open live env along side `https://cloud.jetpack.com/activity-log/:siteSlug`
* Ensure the date range picker still works as it does on live

<img width="843" alt="image" src="https://github.com/user-attachments/assets/0127e92c-5ce7-4378-9d72-ad822ac92cc1">



## Pre-merge Checklist





cc @annacmc
